Automattic/jetpack
# Social: Add upsell nudge for Basic users

We are not promoting the Advanced plan much as of now. This change adds a notice to the editor that should appear once every 3 months after dismissed.

## Proposed changes:

* Added new prop to the `usePublicizeConfig` hook to check if user is on the Basic plan 77f930f6514953690d3f53fb0a620fb72ab240ba
* Added the new upsell notice to the editor 0b7119f59f4cbbdcc6c5f1a0e616c82890dc8602

### Other information:


## Jetpack product discussion


1205003617340866-as-1205091696786993
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* If you have no plans, or have the Advanced plan, you should not see the new notice in the editor. Check this.
* Buy the Basic plan. Now you should see the upsell in the editor below the custom message field.
  * Click the External Link, it should lead you to the Jetpack Cloud pricing page with the Advanced Social modal open
  * Click the close button on the notice. It should dismiss it, and should not come back on refresh.
  * Run `jetpack docker wp option get jetpack_social_dismissed_notices` and check the value of `advanced-upgrade-nudge`. It should be a timestamp, and the value should be 30 days from now. https://www.unixtimestamp.com/
  * Running `jetpack docker wp option delete jetpack_social_dismissed_notices` should bring back the notice. __Note: It brings back the Instagram notice too.__
    * Closing either of the two should work separately not dismissing the other.
